downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -0,0 +1,163 @@
+[id="ref-hub-variables"]
+
+= {HubNameMain} variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description* 
+| *`automationhub_admin_password`* | Required
+| *`automationhub_api_token`* | If upgrading from {PlatformNameShort} 2.0 or earlier, you must either:
+
+- provide an existing {HubNameMain} token as `automationhub_api_token`, or 
+
+- set `generate_automationhub_token` to `true` to generate a new token
+
+Generating a new token will invalidate the existing token.
+| *`automationhub_authentication_backend`* | ldap ?? TBD
```

Comment:
It's not set by default and you'll set to 'ldap' to use LDAP authentication. When this is set to ldap, the follwing variables must also be set:

automationhub_ldap_server_uri
automationhub_ldap_bind_dn
automationhub_ldap_bind_password
automationhub_ldap_user_search_base_dn
automationhub_ldap_group_search_base_dn